src/system/BlocksModule/DependencyInjection/ZikulaBlocksExtension.php

```diff
@@ -1,4 +1,15 @@
 <?php
+/**
+ * Copyright Zikula Foundation 2016 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
```

Comment:
please use the new header from now on. https://github.com/zikula/core/wiki/Coding-Standards#php-files